.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@thorstenmaly Is this file supposed to be added? Please see comment below to drop it.